de-lib/src/main/java/org/iplantc/de/apps/client/AppDetailsView.java

```diff
@@ -82,7 +82,11 @@ void useQuickLaunch(String quickLaunchId,
 
     }
 
-    void load(Presenter presenter);
+    void load(Presenter presenter,
+              final Splittable app,
+              final Splittable appDetails,
+              boolean showQuickLaunchFirst,
+              final String searchText);
 
```

Comment:
Instead of merging the details fetched from service into existing `App` model,  I have updated to code to work with an `App` splittable and separate `details` splittable.